generators/dotnet-templates/README.md

```diff
@@ -58,7 +58,7 @@ The Empty Bot template is the minimal skeleton code for a bot.  It provides a st
 
 
 ## Installation
-1.  Install [.NET Core SDK][4] version 3.1 or higher
+1.  Install [.NET Core SDK][4] version 8.0 or higher
```

Comment:
.NET Core SDK link is "[60]" not "[4]". Check the link references at the end of the document.

Reply:
fixed